### DISCUSSION TITLE
Hybrid ORM Mapping - Attribute/Annotation

### DISCUSSION THREAD

**Discussion by igordeveloper** [doctrine/doctrine2#10535] [Support Questions]
Hi.

There is any way to work with hybrid orm mapping entities? Or the recommended is use the rector and change all entities with attributes?

Thank you.

**Comment by mpdude** [comment]
What exactly do you mean with hybrid - using annotations and attributes at the same time?

**Reply by igordeveloper** [reply]
Yes, some entities with attributes and others with annotations, just a question.

**Reply by mpdude** [reply]
I think it should be possible if the entities are in different namespaces and you configure different mapping drivers for the different directories.

For entities in the same namespace, to my knowledge this is not possible. 

If that helps, you can have attributes and annotations on the same entity class and use either driver to load the configuration.

**Reply by igordeveloper** [reply]
thank you

**Reply by mpdude** [reply]
Thank you for marking the question as resolved ;-)

**Comment by derrabus** [comment]
You might be able to use a `MappingDriverChain` to combine attribute and annotation drivers. Not sure if it's worth it though. Rector should be able to convert your entities in a matter of seconds, allowing you to make a hard switch from annotations to attributes.

**Reply by igordeveloper** [reply]
I hava tried with MappingDriverChain, but have some problems with bundle as Gedmo,
I guess that the best way is use the rector and change all entities(traits) to attributes.

thank you.
